playbooks/edx-west/suclass-prod-app.yml

```diff
@@ -31,6 +31,7 @@
       nginx_sites:
         - lms
         - cms
+        - lms-preview
```

Comment:
I'm pretty sure we don't need it, this is the one that controls generating the nginx conf file